<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deliver_details</class>
 <widget class="QWidget" name="deliver_details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>deliver_details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>收件人：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUser"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPhone"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>邮件标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditEmailTitle"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditContent"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnReturn">
     <property name="text">
      <string>返回主界面</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnTrans">
     <property name="text">
      <string>转发</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>保存文本</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>label</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>textEditContent</zorder>
  <zorder>btnReturn</zorder>
  <zorder>btnTrans</zorder>
  <zorder>btnDelete</zorder>
  <zorder>btnSave</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
